src/components/BarlineGraphComponent.jsx

```diff
@@ -1,208 +1,210 @@
-          },
-          {
-            label: 'Sep'
-          },
-          {
-            label: 'Oct'
-          },
-          {
-            label: 'Nov'
-          },
-          {
-            label: 'Dec'
-          }
-        ]
-      }
-    ],
-    dataset: [
-      {
-        seriesname: 'Cost of Purchase',
-        data: [
-          {
-            value: '16000'
-          },
-          {
-            value: '20000'
-          },
-          {
-            value: '18000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '15000'
-          },
-          {
-            value: '21000'
-          },
-          {
-            value: '16000'
-          },
-          {
-            value: '20000'
-          },
-          {
-            value: '17000'
-          },
-          {
-            value: '25000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '23000'
-          }
-        ]
-      },
-      {
-        seriesname: 'Cost of Damages',
-        renderas: 'line',
-        showvalues: '0',
-        data: [
-          {
-            value: '15000'
-          },
-          {
-            value: '16000'
-          },
-          {
-            value: '17000'
-          },
-          {
-            value: '18000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '19000'
-          },
-          {
-            value: '20000'
-          },
-          {
-            value: '21000'
-          },
-          {
-            value: '22000'
-          },
-          {
-            value: '23000'
-          }
-        ]
-      },
-      {
-        seriesname: 'Loss incurred',
-        renderas: 'area',
-        showvalues: '0',
-        data: [
-          {
-            value: '4000'
-          },
-          {
-            value: '5000'
-          },
-          {
-            value: '3000'
-          },
-          {
-            value: '4000'
-          },
-          {
-            value: '1000'
-          },
-          {
-            value: '7000'
-          },
-          {
-            value: '1000'
-          },
-          {
-            value: '4000'
-          },
-          {
-            value: '1000'
-          },
-          {
-            value: '8000'
-          },
-          {
-            value: '2000'
-          },
-          {
-            value: '7000'
-          }
-        ]
-      }
-    ]
-  }
-  chartConfigs = {
-    id: 'multi_chart',
-    type: 'mscombi2d',
-    width: '45%',
-    height: 400,
-    dataFormat: 'json',
-    dataSource: this.myDataSource,
-  }
-
-  render() {
-    return (
-      <div className="barlineGraph">
-        <Divider hidden />
-        <ReactFC {...this.chartConfigs} />
-      </div>
-    );
-  }
-}
+import React from 'react';
```

Comment:
Please revert the changes to this file to what is in develop as your task doesn't have anything to do with it 🙂